src/inverseRobot.js

```diff
@@ -7,7 +7,24 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  let newKey;
+  let newValue;
+  const result = {};
+
+  for (const key in robot) {
+    newKey = robot[key];
+    newValue = key;
```

Comment:
You can create it inside loop
```suggestion
  const result = {};

  for (const key in robot) {
    const newKey = robot[key];
    const newValue = key;
```

Reply:
i was thinking that is better to create  variable or const outside the loop